### PR TITLE
✨ Add new package specific to Pro features

### PR DESCRIPTION
**Pull request by chuckcarpenter**

### CODE REVIEW

File: packages/pro-js/README [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# @shepherdpro/pro-js
```
Comment (RobbieTheWagner) [inline packages/pro-js/README:1]: Should this be `README.md`?

File: packages/pro-js/src/DataRequest.ts [hunk 2]
```diff
@@ -12,9 +12,9 @@ interface TourStateDb extends DBSchema {
 }
 
 class DataRequest {
-  apiKey: string;
-  apiPath: string;
-  properties?: { [key: string]: unknown };
+  private apiKey: string;
+  private apiPath: string;
+  private properties?: { [key: string]: unknown };
```
Comment (RobbieTheWagner) [inline packages/pro-js/src/DataRequest.ts:17]: The React package was complaining about these being private. Do they need to be? Wondering if using the JS way of making them private would make TS happier? Like `#apiKey` or whatever

**Review by RobbieTheWagner — APPROVED** [review]
More or less looks good to me 👍 . Left a couple comments.

File: packages/pro-js/src/DataRequest.ts (hunk 2 above)
Comment (chuckcarpenter) [reply packages/pro-js/src/DataRequest.ts]: interesting, I don't have the issue b/c the react package isn't reading them, just passing them through. I agree with following correct syntax though.

**Pull request merged**